src/Migrations/MigrationOptions.cs

```diff
@@ -46,5 +58,31 @@ public void AddAssemblyScriptsProvider(Assembly assembly, string resourceNamespa
         /// <param name="folderPath">Path to folder where migration scripts are stored.</param>
         public void AddFileScriptsProvider(string folderPath)
             => AddScriptsProvider(new FileMigrationScriptsProvider(folderPath));
+
+        /// <summary>
+        /// Add action to be executed on database after migration scripts are executed.
+        /// </summary>
+        /// <param name="actionToExecute"></param>
+        public void AddAfterMigrationAction(Func<IDatabase, long, Task> actionToExecute)
+        {
+            _actions.Add(actionToExecute);
+        }
+
+        /// <summary>
+        /// Add action of refreshing all database views.
+        /// </summary>
+        /// <param name="scriptName">Name of the script containing query for refreshing all views. Default one is Kros.KORM\Resources\RefreshViews.sql </param>
+        public void AddRefreshViewsAction(string scriptName = DefaultRefreshViewsScriptName)
```

Comment:
Tento parameter by som tu nedával. Vývojár tam aj tak nemá dať aký iný názov, v danom assembly sa nachádza len tento jeden script. Takže je to len mätúce.